src/test/java/org/fundacionjala/trello/pages/menus/MenuBoards.java

```diff
@@ -48,7 +49,8 @@ public boolean isDisplayed() {
     public BoardPage openPersonalBoard(final String boardName) {
         String personalBoard = String.format(PERSONAL_BOARD, boardName);
         WebElement boardElement = driver.findElement(By.xpath(personalBoard));
-        boardElement.click();
+        action.waitForPageLoadComplete(DELAY);
```

Comment:
I suggest you get times wait from Environment.

Reply:
Done